draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -3745,7 +3729,7 @@ Upgrade: websocket
 <t>
    A "<x:dfn>strong validator</x:dfn>" is representation metadata that changes value whenever
    a change occurs to the representation data that would be observable in the
-   payload body of a <x:ref>200 (OK)</x:ref> response to GET.
+   payload of a <x:ref>200 (OK)</x:ref> response to GET.
```

Comment:
This is a normative change as it's written; is the intent that a change in a strong validator is observable when a header changes, or not?

Reply:
```suggestion
   payload data of a <x:ref>200 (OK)</x:ref> response to GET.
```